test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -10197,6 +10205,26 @@ def forward(self, x):
         loaded_model = onnx.load_from_string(f.getvalue())
         self.assertEqual(loaded_model.graph.output[0].type.tensor_type.shape.dim[1].dim_value, 128)
 
+    @skipIfUnsupportedMinOpsetVersion(10)
+    def test_quantized_linear(self):
+        class M(torch.nn.Module):
+            def __init__(self):
+                super(M, self).__init__()
+                self.fc_1 = torch.nn.quantized.Linear(1, 2)
+
+            def forward(self, x):
+                x = self.fc_1(x)
+                return x
+
+        model = M()
+        input = torch.rand(1, 1)
+        input_tensor = torch.quantize_per_tensor(input, 1, 0, torch.quint8)
+        # Currently, we need convert the model to ScriptModule before export.
+        # The reason is that PackedParams contains int (not tensor).
+        # Then it fails when the exporter calls _trace_and_get_graph_from_model().
+        model = torch.jit.trace(model, input_tensor)
```

Comment:
Does scripting work?
If so, can we write something like:
```python
self.run_test(torch.jit.trace(model, input_tensor), (input_tensor,))
self.run_test(torch.jit.script(model), (input_tensor,))
```

Reply:
Yes.